### ISSUE TITLE
Add a watch option to the cli utility

### ISSUE THREAD

**Issue by peacememories** [rtfeldman/elm-css#180]
Hi, and first of all thanks for making such a useful preprocessor =)

I've recently added [support for additional commands](https://github.com/tomekwi/elm-live/pull/68) to the [elm-live](https://github.com/tomekwi/elm-live) server. The goal was to use elm-css within an auto-building environment.

While this works somewhat (the syntax is not great yet), but I feel a better approach would be to start the live-server and watching builders in parallel. For this, elm-css would of course need to support watching files :)

**Comment by rtfeldman** [comment]
Closing in favor of #327
